### PR TITLE
doc: Add headless-chrome-crawler to example's README

### LINKED ISSUE #1582: (docs proposal) add example tool: headless-chrome-crawler
I have made and have been maintaining a crawler called [headless-chrome-crawler](https://github.com/yujiosaka/headless-chrome-crawler) with the following features:

* Configure concurrency, delay and retry
* Pluggable cache such as [Redis](https://redis.io) to skip duplicate requests
* Pause and resume at any time
* Insert [jQuery](https://jquery.com) automatically
* Priority queue

It's based on Puppeteer, and I think it's a good example making a module using Puppeteer.
If you think it's worth adding to the examples' README, I'll make a small PR for that.

### PR DESCRIPTION
**Pull request by yujiosaka**
close https://github.com/GoogleChrome/puppeteer/issues/1582

@ebidel Let me know if I need to change the description!

### CODE REVIEW

**Comment by yujiosaka** [comment]
I signed it!

File: examples/README.md [hunk 1]
```diff
@@ -31,6 +31,7 @@ const browser = await puppeteer.launch({
 - [Puppetron](https://github.com/cheeaun/puppetron) - Demo site that shows how to use Puppeteer and Headless Chrome to render pages. Inspired by [GoogleChrome/rendertron](https://github.com/GoogleChrome/rendertron).
 - [Thal](https://medium.com/@e_mad_ehsan/getting-started-with-puppeteer-and-chrome-headless-for-web-scrapping-6bf5979dee3e "An article on medium") - Getting started with Puppeteer and Chrome Headless for Web Scraping.
 - [pupperender](https://github.com/LasaleFamine/pupperender) - Express middleware that checks the User-Agent header of incoming requests, and if it matches one of a configurable set of bots, render the page using Puppeteer. Useful for PWA rendering.
+- [headless-chrome-crawler](https://github.com/yujiosaka/headless-chrome-crawler) - Crawler that provides simple APIs to manupluate Headless Chrome and allows you to crawl dynamic websites, powered by Puppeteer.
```
Comment (ebidel) [inline examples/README.md]: you can remove ", powered by Puppeteer." Everything in this list should be powered by pptr :)

**Review by ebidel — CHANGES_REQUESTED** [review]

File: examples/README.md (hunk 1 above)
Comment (yujiosaka) [reply examples/README.md]: @ebidel Thanks! I just pushed the fix!

**Review by ebidel — APPROVED** [review]

**Comment by ebidel** [comment]
Can you confirm that you signed the CLA? The bot is having troubles finding you.

**Comment by yujiosaka** [comment]
@ebidel Sorry, it was my fault that I was using a different email address. I modified my commit's author and commuter, and it should be fine now!

**Comment by ebidel** [comment]
Ah, fixed itself :)

**Pull request merged**